mp4parse/src/lib.rs

```diff
@@ -738,12 +738,6 @@ pub struct MediaContext {
     pub userdata: Option<Result<UserdataBox>>,
 }
 
-impl MediaContext {
-    pub fn new() -> MediaContext {
-        Default::default()
-    }
-}
-
```

Comment:
This could've been removed in https://github.com/mozilla/mp4parse-rust/commit/c196ca8e47a5df439b69c17df6ebd1ed59ed053d, but better late than never